src/Ledger/Ratify.lagda

```diff
@@ -294,5 +533,24 @@ data _⊢_⇀⦇_,RATIFY'⦈_ : RatifyEnv → RatifyState → GovActionID × Gov
 _⊢_⇀⦇_,RATIFY⦈_ : RatifyEnv → RatifyState → List (GovActionID × GovActionState) → RatifyState → Set
 _⊢_⇀⦇_,RATIFY⦈_ = SS⇒BS (λ where (Γ , _) → Γ ⊢_⇀⦇_,RATIFY'⦈_)
 \end{code}
+} %% end small
 \caption{RATIFY transition system}
+\label{fig:ratify-transition-system}
 \end{figure*}
+Finally, Figure~\ref{fig:ratify-transition-system} defines three functions: \AgdaFunction{RATIFY-Accept}, \AgdaFunction{RATIFY-Reject}, and \AgdaFunction{RATIFY-Continue}, along with the syntax \verb!_⊢_⇀⦇_,RATIFY⦈_!.
+The syntax \verb!_⊢_⇀⦇_,RATIFY⦈_! is a wrapper around another function, represented by \AgdaDatatype{SS⇒BS}, which is a transformation from some source state (\AgdaDatatype{SS}) to a binary state (\AgdaDatatype{BS}).
```

Comment:
No, `SS⇒BS` is a small-step to big-step transformer, it even says to in a comment next to its definition. It's basically the transitive closure, except that it also keeps track of an index. `RATIFY'` defines how to ratify one action, and `RATIFY` then just ratifies a list of actions.